purview/models/__init__.py

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from purview.models.koji import KojiBuild, KojiTask, KojiTag
+from purview.models.bugzilla import BugzillaBug
+from purview.models.distgit import DistGitRepo, DistGitPush, DistGitBranch, DistGitCommit
+from purview.models.errata import Advisory, AdvisoryState
+from purview.models.freshmaker import FreshmakerEvent, ContainerBuilds
+from purview.models.user import User
+
+all_models = {KojiBuild, KojiTask, KojiTag, BugzillaBug, DistGitRepo, DistGitPush,
```

Comment:
Bike-shedding: Could this be alphabetized? That would make an error message to the user easier to read if we list out all the valid models.